img_cutter.py

```diff
@@ -0,0 +1,70 @@
+import cv2
+import json
+import argparse
+from pathlib import Path
+
+from f2b import F2B, draw
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--img', help='path of image file', default='examples/seoul-station_4K.png', type=str)
+parser.add_argument('--out_dir', help='path of output directory', default='cvat_images', type=str)
+parser.add_argument('--f2b_width', help='f2b max_inference_width', default=1333, type=int)
+parser.add_argument('--f2b_height', help='f2b max_inference_height', default=1333, type=int)
+parser.add_argument('--f2b_x', help='f2b overlapx_px', default=100, type=int)
+parser.add_argument('--f2b_y', help='f2b overlapy_px', default=100, type=int)
+parser.add_argument('--out_f2b', help='path of output filename for f2b settings', default='f2b_set.json', type=str)
```

Comment:
can do the arguments as direct filepaths instead of just filename? "path to output file for f2b settings"